mkosi/resources/man/mkosi.1.md

```diff
@@ -1932,6 +1938,41 @@ config file is read:
     each individual subimage as if they were "universal" settings. See
     the **Building multiple images** section for more information.
 
+### [Addon] Section
```

Comment:
I think we should name this section `PeAddon` and the other `UkiProfile` (or `UnifiedKernelImageProfile`), it's more symmetrical with the options and doesn't give up the short names to these more specialised uses (also in the case of the Profile section it could be confusing with our profiles).